src/hooks/useCourt.js

```diff
@@ -160,14 +178,22 @@ export function useDisputeActions() {
   return { draft, commit, reveal, leak, appeal, confirmAppeal }
 }
 
-export function useCourtSubscription(courtAddress) {
-  const [result] = useSubscription({
-    query: CourtConfigSubscription,
-    variables: { id: courtAddress },
-  })
+export function useTotalActiveBalancePolling(termId) {
+  const POLL_EVERY = 1000
+
+  const jurorRegistryContract = useJurorRegistryContract()
+  const [totalActiveBalance, setTotalActiveBalance] = useState(bigNum(-1))
+
+  const fetchTotalActiveBalance = useCallback(
+    cancelled => {
+      jurorRegistryContract.totalActiveBalanceAt(termId).then(balance => {
+        if (!cancelled) setTotalActiveBalance(balance)
+      })
+    },
+    [jurorRegistryContract, termId]
+  )
 
-  // TODO: handle possible errors
-  const courtConfig = result.data && result.data.courtConfig
+  useInterval(fetchTotalActiveBalance, POLL_EVERY, true)
```

Comment:
I am wondering if we shouldn’t use a `setTimeout()` instead? With `setInterval()`, every request taking more than one second will get cancelled, but we might want to wait for a request to complete before starting the next one. What do you think?

Reply:
Yes very good point